docs/reference/administration/system-options: add options for pki-certs (#278)
canonical/snap-docs: docs/reference/administration/system-options.md

> ⓘ Available only on Ubuntu Core.

## system pki.certs.custom

> A custom certificate named `<name>` can be managed with the following system options:

Reviewer: ```suggestion
Enables custom certificate management with the following options (replace `<name>` with the certificate name):
```
